src/cli/cli.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef __CLI_HPP__
+#define __CLI_HPP__
+
+#include "task.hpp"
+
+
+typedef const struct CLI_menu_
+{
+    char* cmd;
+    const char *const fnName; // menu function name
```

Comment:
Why are these so different?

`cmd` probably should be `const char*` since we expect a string literal.  Since these are also expected to be compile time literals, `const char* const` isn't a terrible choice either.  In general, for pointers, we prefer the asterisk to be adjacent to the type.

Reply:
Fixed